File: emulinker/src/main/java/org/emulinker/kaillera/master/PublicServerInformation.java
package org.emulinker.kaillera.master;

import org.emulinker.config.RuntimeFlags;

public class PublicServerInformation {
  private final RuntimeFlags flags;

  public PublicServerInformation(RuntimeFlags flags) {
    this.flags = flags;
  }

  public String getServerName() {
    return flags.serverName();
  }

  public String getLocation() {
    return flags.serverLocation();
  }

  public String getWebsite() {
    return flags.serverWebsite();
  }

  public String getConnectAddress() {
    return flags.serverAddress();
  }
}

File: emulinker/src/main/java/org/emulinker/kaillera/master/PublicServerInformation.kt
package org.emulinker.kaillera.master

import org.emulinker.config.RuntimeFlags

class PublicServerInformation(flags: RuntimeFlags) {
  val serverName: String = flags.serverName
  val location: String = flags.serverLocation
  val website: String = flags.serverWebsite
  val connectAddress: String = flags.serverAddress
}
